app/models/fact_value.rb

```diff
@@ -81,13 +81,16 @@ def self.count_each(fact, options = {})
     output
   end
 
-  def self.build_facts_hash(facts)
+  def self.build_facts_hash(facts, add_host)
```

Comment:
In order to allow this method to be backwards compatible (or if any plugin is using it, making sure it still works), please change it to

```ruby
def self.build_facts_hash(facts, add_host=true)
```

IMO `include_hostname` is a better name for it but don't take my word for it..